### PR TITLE
Add booking availability to event and place json-ld

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Added `bookingAvailability` to event, subEvent, and place JSON-LD

---

Note: Some things got a bit shuffled around in the diffs due to the Stoplight Studio app which for some reason sometimes moves some properties around.

Also note that at the moment event and place don't re-use each other's models but have their own models for their properties. This makes it easier to tweak the descriptions and some details where they slightly differ.

To preview, go to https://docs.publiq.be, login with `apidocs@publiq.be` and go to the UiTdatabank project. Switch the branch from "Stable" to "booking-availability".

### CODE REVIEW

**Comment by bertramakers** [comment]
Converted this to a draft so we can use this to group all other PRs for the bookingAvailability feature into, so we can publish them all at once when the feature is ready and deployed.

File: models/event-subEvent.json [hunk 1]
```diff
@@ -2,6 +2,33 @@
   "title": "event.subEvent",
   "type": "array",
   "description": "A list of occurences of the [event](./event.json) when using the `single` or `multiple` [calendarType](./event-calendarType.json).",
+  "x-examples": {
```
Comment (LucWollants) [inline models/event-subEvent.json:5]: Should these examples also include a `bookingAvailability` property?

Comment (bertramakers) [reply models/event-subEvent.json:5]: Yes good that you spotted this! I've added them in 38b0b7b

File: reference/entry.json [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
...Availability  |\n| periodic  | startDate, endDate  | openingHours, status, bookingAvailability  |\n| permanent  |   | openingHours, status, bookingAvailability  |\n\n<!-- theme: warning -->\n\n> If `status` or `bookingAvailability` is missing on the event or a subEvent, it will default to `Available`.\n>\n> This means that if the event or subEvent(s) have an existing `status` or `bookingAvailability` that is not `Available`, and you do not include this `status` or `bookingAvailability` in the new calendar information, they will get reverted back to the default `Available`!\n\n<!-- theme: success -->\n\n> You can use `single` and `multiple` interchangeably as long as you also include `subEvent` as well. The API will use the correct type based on the number of subEvents inside `subEvent`.\n\n<!-- theme: info -->\n\n> This endpoint also supports a deprecated schema that uses `timeSpans` instead of `subEvent`. The `timeSpans` also have a slightly different structure than `subEvent`. For new integrations, it is recommended to use the schema with the `subEvent` property. \n> \n> For existing integrations the `timeSpans` property will be supported indefinitely for backward compatibility."
```
Comment (LucWollants) [inline reference/entry.json:324]: Maybe a small change for

```
This means that if the event or subEvent(s) have an existing status or bookingAvailability that is not Available, and you do not include this status or bookingAvailability in the new calendar information, they will get reverted back to the default Available!
```

to

```
Although the status and bookingAvailability are optional they have a default value of `Available`. When the status and bookingAvailability is not provided an already set value will be overwritten to `Available`!
```

Comment (bertramakers) [reply reference/entry.json:324]: Perfect! 👌

**Pull request merged**